### PR TITLE
Updated BungeeCord docs

### PR DESCRIPTION
**Pull request by phit**
As it has been years at this point and the PR's have no progress,
I'd rather link to an actual solution to people, tired of having to link
people a solution when it could just be in the Docs.

Also moved without IP Forwarding to the end as nobody should be really using that, and with is more important.

### CODE REVIEW

File: source/server/getting-started/bungeecord.rst [hunk 1]
```diff
@@ -27,37 +27,40 @@ IP Forwarding
...
-This will work with all implementations of Sponge, including with mods.
 
 Using BungeeCord with IP Forwarding
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If you wish to use IP Forwarding:
 
 * In the BungeeCord ``config.yml``, set ``ip_forward`` to ``true``
-* In Sponge's config (config/sponge/global.conf), set ``modules.bungeecord`` to ``true`` and ``bungeecord.ip-forwarding`` to
+* In Sponge's config (``config/sponge/global.conf``), set ``modules.bungeecord`` to ``true`` and ``bungeecord.ip-forwarding`` to
```
Comment (ST-DDT) [inline source/server/getting-started/bungeecord.rst]: Please limit the line length to 120 characters. (split before ``bungeecord.ip-forwarding``)

File: source/server/getting-started/bungeecord.rst [hunk 2]
```diff
@@ -27,37 +27,40 @@ IP Forwarding
...
   ``true``
 * If you have any other server software, consult the documentation for that server.
 
 This must be done for **all** servers that are connected to the BungeeCord network. Then, just follow the instructions
 for using BungeeCord without IP Forwarding.
+
+
```
Comment (ST-DDT) [inline source/server/getting-started/bungeecord.rst]: Duplicate empty line

File: source/server/getting-started/bungeecord.rst [hunk 3]
```diff
@@ -27,37 +27,40 @@ IP Forwarding
...
+ `Waterfall <https://github.com/WaterfallMC/Waterfall/blob/master/README.md#waterfall->`_, or a BungeeCord plugin like
+ `SpongePls <https://forums.spongepowered.org/t/spongepls/9891>`_. SpongePowered does not officially support these
+ products, although they are developed by trusted members of the community.
 
-While it is recommended that you use IP Forwarding wherever possible. If you do not wish to do so, simply ensure that
-``online-mode`` is set to ``false`` in your ``server.properties`` file and add the server details to Bungee's
-``config.yml`` file. Bungee will then forward any connections to the server when required. It is a good precaution to
-set the ``server-port`` to something other than ``25565``.
+For SpongeVanilla or Vanilla clients connecting to unmodded SpongeForge, IP Forwarding works out of the box.
+`Pull <https://github.com/SpigotMC/BungeeCord/pull/1557>`_ `requests <https://github.com/SpigotMC/BungeeCord/pull/1678>`_
```
Comment (ST-DDT) [inline source/server/getting-started/bungeecord.rst]: I would go for `Pull requests: #1557 and #1678` so you actually see that there are more than one PRs releated to that, (but thats optional IMO)
Also watch the line length ( 121 > 120 ;-) ).

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: source/server/getting-started/bungeecord.rst (hunk 3 above)
Comment (phit) [reply source/server/getting-started/bungeecord.rst]: the numbers mean nothing to readers, and you can see it's two links as the blue line of a link does not include the space between

File: source/server/getting-started/bungeecord.rst (hunk 1 above)
Comment (phit) [reply source/server/getting-started/bungeecord.rst]: it was already above 120 qq

**Review by ST-DDT — APPROVED** [review]
I cannot test the technical details of this, but otherwise it looks good to me.

**Comment by ST-DDT** [comment]
Why does Travis CI not try to build this PR? It works (at least locally for me).

**Comment by vectrixdevelops** [comment]
@ST-DDT It's apparently listed as Abuse on Travis, @phit is going to try contact their support to clear that up.

**Comment by kashike** [comment]
@phit was caught stealing cookies from their cookie jar. :cookie:

**Comment by dualspiral** [comment]
I don't think SpongePls is maintained any more and does not support negative packet IDs that some mods like to use. I think we should just be suggesting Waterfall for simplicity (other forks are available).

Otherwise, I have no interest in chasing up the Bungee PRs any further so I agree with the removal of those statements.

**Comment by phit** [comment]
Waterfall has issues with mods too, which come from upstream..
https://github.com/WaterfallMC/Waterfall/issues/136 
https://github.com/HexagonMC/BungeeCord/issues/148
https://github.com/WaterfallMC/Waterfall/issues/216

both of which I had to "fix" for my server [1](https://github.com/Stonebound/BungeeCord/commit/00b5d0bf6c62cc7656a0cf39844a52279e6bd5d9)  [2](https://github.com/Stonebound/BungeeCord/commit/2d1ad7d77cc70b2073b7bd92b608edfd5066c8d6)
so really it doesn't matter, both options have issues

File: source/server/getting-started/bungeecord.rst [hunk 4]
```diff
@@ -27,37 +27,39 @@ IP Forwarding
...
+  ``bungeecord.ip-forwarding`` to ``true``
 * If you have any other server software, consult the documentation for that server.
 
 This must be done for **all** servers that are connected to the BungeeCord network. Then, just follow the instructions
 for using BungeeCord without IP Forwarding.
+
+Using BungeeCord without IP Forwarding
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While it is recommended that you use IP Forwarding wherever possible. If you do not wish to do so, simply ensure that
```
Comment (Inscrutable) [inline source/server/getting-started/bungeecord.rst]: Broken clause. Delete the While.

Comment (phit) [reply source/server/getting-started/bungeecord.rst]: I didn't even write this, but done..

**Pull request merged**